include/miopen/config.h.in

```diff
@@ -49,6 +49,7 @@
 #cmakedefine01 MIOPEN_ENABLE_SQLITE_BACKOFF
 #cmakedefine01 MIOPEN_USE_MLIR
 #cmakedefine01 MIOPEN_USE_COMPOSABLEKERNEL
+#cmakedefine01 MIOPEN_ENABLE_AI_HEUR
```

Comment:
[Recommendation] Maybe 
```suggestion
#cmakedefine01 MIOPEN_ENABLE_AI_TUNING_HEUR
```

Reply:
I was hoping this cmake variable would be used to toggle both tuning model as well as the solver prediction model.

Reply:
Most likely we'll need to control these separately. At least, for A/B testing.

Reply:
Maybe even
```
MIOPEN_ENABLE_AI_KERNEL_TUNING
MIOPEN_ENABLE_AI_IMMEDIATE_MODE_FALLBACK
```
for this PR and for #2055? I really believe we need separate controls.

Reply:
Changed it's name to MIOPEN_ENABLE_KERNEL_TUNING

Reply:
Resolved